client/components/tinymce/plugins/contact-form/dialog/settings.jsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+export default React.createClass( {
+	displayName: 'ContactFormDialogFormSettings',
+
+	propTypes: {
+		to: PropTypes.string,
+		subject: PropTypes.string,
+		onUpdateSettings: PropTypes.func.isRequired
+	},
+
+	render() {
+		return (
+			<div className="editor-contact-form-modal-settings">
+				<SectionHeader label={ this.translate( 'Contact Form Notification Settings' ) } />
+				<Card>
+					<p>
+						{ this.translate( 'If you don’t make any changes here, feedback will be sent to the author of the page/post and the subject will be the name of this page/post.' ) }
```

Comment:
Elsewhere in the editor, we've tried to create unique strings depending on the post type, rather than imposing the burden of differentiating "post/page" on the user.

May be worth exploring whether it's possible to provide the post type as an argument to the string translation. Having a generalized approach here will be good for when we add support for custom post types.

Could you see if this works?

```js
this.translate( '...feedback will be sent to the author of the %(postType)s', { args: { postType: this.translate( 'post' ) } } );
```